backend/ChessClubRegistration/src/main/java/com/marius/lt/ChessClubRegistration/entities/enums/Genders.java

```diff
@@ -0,0 +1,27 @@
+package com.marius.lt.ChessClubRegistration.entities.enums;
+
+public enum Genders {
+    MALE("male", 3, 5),
+    FEMALE("female", 4, 6);
+
+    private final String gender;
```

Comment:
Smulkmena, bet si lauka vadiciau `title` arba `name`, nes pamates toki koda, nelabai galeciau nuspet koks bus jo rezultatas:
`Genders.MALE.gender -> ?? `

Matant taip:
`Genders.MALE.title -> `
Butu paprasciau nuspet, kad to rezultatas bus MALE lyties pavadinimas.